docs/why_mf_next.md

```diff
@@ -108,23 +125,16 @@ localization flow:
 
 ### 4. Designed to be API only, plain text, UI, “imperative style”
 
-The main (only?) use case for `MessageFormat` is: load the string from resources,
+The typical use case for `MessageFormat` is: load the string from resources,
 replace placeholders, and return the string result with placeholders replaced. \
 An i18n-aware `printf`, basically.
 
-It does not play well with binding, formatting tags (think `html`),
+It does not account for formatting tags (think `html`),
 or “document-like” content (for example templating systems like
 [freemarker](https://freemarker.apache.org/),
 [mustache](https://mustache.github.io/), even JSP, PHP, etc.)
 
-Because it is API only it has no standard way to store the stings in a
-serialized format and to carry info or directives for translators or
-localization tools. \
-So there is no way for a message to reference another message, or to fallback
-to a different locale. That is all left to the "host resource manager"
-(whatever that is for the given tech stack)
```

Comment:
I agree about the first part.

I'm not sure I agree about the second part, which is a description of resource management and/or resource files. There are lots of formatter adjacent things that are not in scope for MF2 itself (and which MF1 never pretended to solve). I can't use 'suggestion' to restore text, so will separately commit.

Reply:
I think the first can be restored, but should be changed. There is a standard string serialization defined by the API, used all the time. The point to make is that it doesn't the kinds of information needed, so people have developed non-interoperable wrappers for it.
OLD
Because it is API only it has no standard way to store the stings in a
serialized format and to carry info or directives for translators or
localization tools.
NEW
The format defined by the API provides no standard way to carry the necessary info or directives for translators or localization tools, so implementations have had to develop their own, with no interoperability.

Reply:
I will add your text.

Reply:
Thanks